etl/command.py

```diff
@@ -248,16 +241,15 @@ def main_cli(
         config.SUBSET = subset
 
     kwargs = dict(
-        steps=steps,
+        includes=steps,
         dry_run=dry_run,
         force=force,
         private=private,
         grapher=grapher,
         export=export,
-        downstream=downstream,
         only=only,
         exact_match=exact_match,
-        exclude=exclude,
+        excludes=exclude.split(",") if exclude else None,
```

Comment:
I suppose `include` and `exclude` would be preferrable (shorter, imperative) than `includes` and `excludes`.

Reply:
(same in other similar places)